main.py

```diff
@@ -118,9 +313,56 @@ def disable_pipeline(self, request: Request) -> JSONResponse:
             log.debug(f"disable_pipeline result {msg} 404")
             raise HTTPException(404, detail=msg)
         msg = f"Pipeline {pipeline_id} disabled"
+        self.enable_basic_pipeline()
         log.debug(f"disable_pipeline result {msg} 200")
         return JSONResponse(msg)
 
+    def enable_basic_pipeline(self):
+        """Return to basic pipeline
+        After disabling a pipeline"""
+        content = {
+            'of_lldp': {'base': 0},
+            'coloring': {'base': 0},
+            'mef_eline': {'epl': 0, 'evpl': 0},
+        }
+        enabled_napps = self.get_enabled_napps()
+        self.required_napps = enabled_napps.intersection(self.subscribed_napps)
+        self.emit_event('enable_table', content=content)
+
+    @listen_to("kytos/flow_manager.flow.added")
+    def on_flow_mod_added(self, event):
+        """Looking for recently added flows"""
+        self.handle_flow_mod_added(event)
+
+    def handle_flow_mod_added(self, event):
+        """Handle recently added flows"""
+
+    @listen_to("kytos/of_core.handshake.completed")
+    def on_handshake_completed(self, event):
+        """Listen to new switches added"""
+        self.handle_handshake_completed(event)
+
+    def handle_handshake_completed(self, event):
+        """Handle new added switches"""
+        switch_id = event.content['switch'].dpid
+        pipeline = self.get_enabled_table()
+        if pipeline.get('status') is not None:
+            self.install_miss_flows(pipeline, switch_id)
```

Comment:
Error 424 (SwitchNotConnectedError) is occurring because `of_multi_table` tries to install miss flow entries when `of_core` listens to the `"kytos/of_core.handshake.completed"` event. Adding a `sleep(5)` before sending the miss flow entries solves the problem.

Reply:
Deleted